modules/mtl/arrow-mtl-data/src/main/kotlin/arrow/mtl/typeclasses/Composed.kt

```diff
@@ -97,20 +95,17 @@ interface ComposedTraverse<F, G> :
   companion object {
     operator fun <F, G> invoke(
       FF: Traverse<F>,
-      GF: Traverse<G>,
-      GA: Applicative<G>
```

Comment:
This was not used and overcomplicated some function signatures using it.
Removing it had no effect as far as I can tell